src/pages/Team/Team.js

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import constTeam from './ConstTeam';
 
 function Team() {
-  return <h1 className="mx-auto text-center text-7xl">Team</h1>;
+  return (
+    <div >
+      <h1 className="mx-auto  text-5xl  py-16 px-10 lg:px-40">  
```

Comment:
suggest to add the following
className="pt-5 text-3xl md:text-5xl format-normal leading-normal"

Reply:
[](url)
![result](https://user-images.githubusercontent.com/36269806/209658468-9ff21499-2ed5-4d65-90d4-8a05f2f59d8a.PNG)
Danah this is how it look when I apply your suggestion to it